spec/factories/users.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :user do
-    email   "mail@mail.ru"
-    password "secret"
+    email                 "mail@mail.ru"
+    password              "secret"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.